google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineRuntimeInstance.java

```diff
@@ -87,6 +89,38 @@ public void run(@NotNull ProgressIndicator indicator) {
     }
   }
 
+  /**
+   * Disambiguates running deployment line items by prepending a timestamp. Also appends the
+   * cloud project and version id's to the deployment string.
+   */
+  @NotNull
+  @Override
+  public String getDeploymentName(@NotNull DeploymentSource source,
+      AppEngineDeploymentConfiguration configuration) {
+    String deploymentName = String.format("[%s] ", DateTime.now().toString("yyyy-MM-dd HH:mm:ss"));
+
+    // If its a user specified archive source then we want to label it by the name of the archive
+    if (source instanceof UserSpecifiedPathDeploymentSource && source.getFile() != null) {
+      deploymentName += source.getFile().getName();
```

Comment:
I must be missing something, but how is the space being added here between the timestamp and the deployable name?